### PR TITLE
Fix Timesheet employee query to show names

### PR DESCRIPTION
**Pull request by bcornwellmott**
The current Employee lookup only shows the employee number. This fix will also show the employee name to make it easy for employees to find their name.

### CODE REVIEW

File: erpnext/projects/doctype/timesheet/timesheet.py [hunk 1]
```diff
@@ -293,7 +293,7 @@ def get_activity_cost(employee=None, activity_type=None):
 
 @frappe.whitelist()
 def get_employee_list(doctype, txt, searchfield, start, page_len, filters):
-	return frappe.db.sql("""select distinct(employee) as employee
+	return frappe.db.sql("""select distinct employee, employee_name
 		from `tabSalary Structure` where salary_slip_based_on_timesheet=1
 		and employee like %(txt)s limit %(start)s, %(page_len)s""", 
-		{'txt': "%%%s%%"%(txt), 'start': start, 'page_len': page_len})
+		{'txt': "%%%s%%"% txt, '_txt': txt.replace("%", ""), 'start': start, 'page_len': page_len})
```
Comment (rohitwaghchaure) [inline erpnext/projects/doctype/timesheet/timesheet.py]: @bcornwellmott 
What is use of _txt?

File: erpnext/projects/doctype/timesheet/timesheet.py [hunk 2]
```diff
@@ -293,7 +293,7 @@ def get_activity_cost(employee=None, activity_type=None):
 
 @frappe.whitelist()
 def get_employee_list(doctype, txt, searchfield, start, page_len, filters):
-	return frappe.db.sql("""select distinct(employee) as employee
+	return frappe.db.sql("""select distinct employee, employee_name
 		from `tabSalary Structure` where salary_slip_based_on_timesheet=1
 		and employee like %(txt)s limit %(start)s, %(page_len)s""",
```
Comment (rohitwaghchaure) [inline erpnext/projects/doctype/timesheet/timesheet.py]: Add (employee like %(txt)s or employee_name like %(txt)s)

File: erpnext/projects/doctype/timesheet/timesheet.py (hunk 1 above)
Comment (bcornwellmott) [inline erpnext/projects/doctype/timesheet/timesheet.py]: Honestly, not sure. I duplicated what was on Salary Structure.

File: erpnext/projects/doctype/timesheet/timesheet.py (hunk 2 above)
Comment (bcornwellmott) [inline erpnext/projects/doctype/timesheet/timesheet.py]: Will change

**Comment by bcornwellmott** [comment]
@rohitwaghchaure, the fixes you requested should be complete.

**Pull request merged**